packages/mds-geography/tests/api.spec.ts

```diff
@@ -0,0 +1,468 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+// eslint directives:
+/* eslint-disable no-plusplus */
+/* eslint-disable no-useless-concat */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable promise/prefer-await-to-callbacks */
+
+/* eslint-reason extends object.prototype */
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+import should from 'should'
+import sinon from 'sinon'
+import supertest from 'supertest'
+import test from 'unit.js'
+import db from '@mds-core/mds-db'
+import uuid from 'uuid'
+import { Geography } from '@mds-core/mds-types'
+import { NotFoundError } from '@mds-core/mds-utils'
+import { ApiServer } from '@mds-core/mds-api-server'
+import {
+  POLICY_UUID,
+  GEOGRAPHY_UUID,
+  GEOGRAPHY2_UUID,
+  LA_CITY_BOUNDARY,
+  DISTRICT_SEVEN,
+  SCOPED_AUTH
+} from '@mds-core/mds-test-data'
+import { api } from '../api'
+
+/* eslint-disable-next-line no-console */
+const log = console.log.bind(console)
+
+const request = supertest(ApiServer(api))
+
+const APP_JSON = 'application/json; charset=utf-8'
+const EMPTY_SCOPE = SCOPED_AUTH([], '')
+const EVENTS_READ_SCOPE = SCOPED_AUTH(['events:read'])
+const POLICIES_WRITE_SCOPE = SCOPED_AUTH(['policies:write'])
+const POLICIES_READ_SCOPE = SCOPED_AUTH(['policies:read'])
+const POLICIES_DELETE_SCOPE = SCOPED_AUTH(['policies:delete'])
+const sandbox = sinon.createSandbox()
+
+describe('Tests app', () => {
+  describe('Geography endpoint tests', () => {
```

Comment:
I don't see a test case that exercises `get_read_only === true`, but I might be missing something?